docs/content/learn/tutorials/control/basic-profile.md

```diff
@@ -0,0 +1,1282 @@
+      should be defined.
+  - id: s1.1.1_gdn.2
+    name: item
+    prose: Individuals with allocated information security responsibilities may delegate
+      security tasks to others. Nevertheless they remain accountable and should determine
+      that any delegated tasks have been correctly performed.
+  - id: s1.1.1_gdn.3
+    name: item
+    prose: |
+      Areas for which individuals are responsible should be stated. In particular the following should take place:
+
+      1. the assets and information security processes should be identified and defined;
+      1. the entity responsible for each asset or information security process should be assigned and the details of this responsibility should be documented;
+      1. authorization levels should be defined and documented;
+      1. to be able to fulfil responsibilities in the information security area the appointed individuals should be competent in the area and be given opportunities to keep up to date with developments;
+      1. coordination and oversight of information security aspects of supplier relationships should be identified and documented.
+```
+
+As profile authors, let's say we'd like to insert a new `part` after `s.1.1.1_gdn.3` with additional guidance on setting an automatic lock on a specific platform:
+
+```yaml {linenos=table}
+---
+alters:
+- control-id: s1.1.1
+  adds:
+  - by-id: s1.1.1_gdn.3
+    position: after
+    parts:
+    - id: s1.1.1_gdn.4
+      name: item
+      prose: 'Users of devices running Gnome can adjust the inactivity timeout using
+        the following link: https://help.gnome.org/admin/system-admin-guide/stable/desktop-lockscreen.html.en'
+```
+
+A `modify` object has an optional array of `alters` objects.
+`alters` objects select a single control through the mandatory `control-id` field and modifies them with the following sub-objects:
+* `adds` is an array of objects can add some content to a control at a position specified by the `position` field, which can be "before", "after", "starting", and "ending".
+  If `position` is set to "before" or "after", an object must be selected via the `by-id` field.
+
+* `removes` is an array of objects that can remove some content from a control as selected by the `by-class`, `by-id`, `by-item-name`, `by-name`, or `by-ns` fields.
+{{% /tab %}}
+{{< /tabs >}}
+
+## The Final Profile
+
+After applying all of the techniques discussed in this tutorial, we obtain an OSCAL profile with the following structure:
+
+{{< tabs XML JSON YAML >}}
+{{% tab %}}
+```xml {linenos=table}
+<?xml version="1.0" encoding="UTF-8"?>
+<profile xmlns="http://csrc.nist.gov/ns/oscal/1.0" uuid="c0dc468c-934e-4fe9-b5bf-9fc63f5a2915">
+  <metadata>
+    <title>Sample Security Profile 
+      <em>for Demonstration</em> and Testing
+    </title>
+    <last-modified>2023-04-10T10:31:28.355446-04:00</last-modified>
+    <version>1.0</version>
+    <oscal-version>1.04</oscal-version>
```

Comment:
Blocking: `1.04` is not an official OSCAL version for this example and other real-world data, but `1.0.4` is.
```suggestion
    <oscal-version>1.0.4</oscal-version>
```